extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -190,7 +190,11 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     this.element.setAttribute('data-amp-slot-index',
         this.win.ampAdSlotIdCounter++);
     this.lifecycleReporter_ = this.initLifecycleReporter();
-    this.ampAnalyticsConfig_ = null;
+    if (this.ampAnalyticsConfig_) {
+      this.ampAnalyticsConfig_.parentElement.removeChild(
```

Comment:
I don't think this will work.  This is a JSON object as opposed to an actual element.  Instead you need to store the result of insertAnalyticsElement (modify to ensure it returns reference to amp analytics element).  Did you test this on a page running local gulp?